test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature

```diff
@@ -65,3 +65,14 @@ Feature: Status Desktop community
             | new_community_name       | new_community_description  | new_community_color |
             | myCommunityNamedChanged  | Cool new description 123   | #ff0000             |
 
+Scenario Outline: Admin deletes a community channel
```

Comment:
@stefandunca it doesn't need to be `Scenario Outline`, just `Scenario` is enough here.

Reply:
Indeed. 
Also, I would split this scenario in two so that we have one scenario specifically to delete and one to test the cancel.
If you want, you can keep this one, but in that case, it should be names "Admin cancels a community channel delete, but then deletes a community channel for real" (it's not a pretty name lol)

Reply:
> @stefandunca it doesn't need to be Scenario Outline, just Scenario is enough here.

I see, `Scenario Outline` is used with `Examples` to have multiple test data inputs.

> Also, I would split this scenario in two so that we have one scenario specifically to delete and one to test the cancel.
If you want, you can keep this one, but in that case, it should be names "Admin cancels a community channel delete, but then deletes a community channel for real" (it's not a pretty name lol)

:) that was an attempt to increase the coverage and validate the previous . However, I don't think it brings benefits to worth a separate test case. I will drop the "the user attempts to deletes current channel but cancels"